exercises/sum-of-multiples/example.lua

```diff
@@ -0,0 +1,24 @@
+return function(numbers)
+  numbers = numbers or { 3, 5 }
+
+  return {
+    to = function(limit)
```

Comment:
might be hard for users new to lua to grasp this chaining pattern of named returns